tests/unit/nupic/encoders/logenc_test.py

```diff
@@ -255,24 +259,66 @@ def testInitWithN(self):
     n = 100
     le = LogEncoder(n=n, forced=True)
     self.assertEqual(le.encoder.n, n)
-  
-  
+
+
   def testMinvalMaxVal(self):
     '''
     Verifies unusual instances of minval and maxval are handled properly
     '''
-    
+
     self.assertRaises(ValueError, LogEncoder, n=100, minval=0, maxval=-100, forced=True)
     self.assertRaises(ValueError, LogEncoder, n=100, minval=0, maxval=1e-07, forced=True)
-    
+
     le = LogEncoder(n=100, minval=42, maxval=1.3e12, forced=True)
-    
-    expectedRadius = 0.552141792732 
+
+    expectedRadius = 0.552141792732
     expectedResolution = 0.110428358546
     self.assertAlmostEqual(le.encoder.radius, expectedRadius)
     self.assertAlmostEqual(le.encoder.resolution, expectedResolution)
 
 
+  def testReadWrite(self):
+    le = LogEncoder(w=5,
```

Comment:
`encoder1` and then `encoder2` for the deserialized one? `le` doesn't seem very descriptive.